explainer.md

```diff
@@ -161,119 +150,107 @@ user agent.
 
 ## Usage examples
 
-The simplest possible usage of the API is a single call to `Badge.set` from a
-foreground context (which might be used to show an unread count in an email
-app):
+The API is divided into two parts: one for setting/clearing a badge on the
+current document, and one for setting/clearing a badge on the current
+application.
+
+To simply set a numeric badge on the current document:
 
 ```js
-Badge.set(getUnreadCount());
+navigator.setClientBadge(getUnreadCount());
 ```
 
-This will set the badge for all pages and apps in the current origin until it is
-changed. If `getUnreadCount()` (the argument to `Badge.set`) is 0, it will
+If `getUnreadCount()` (the argument to `navigator.setClientBadge`) is 0, it will
 automatically clear the badge.
 
 If you just want to show a status indicator flag without a number, use the
-Boolean mode of the API by calling `Badge.set` without an argument, and
-`Badge.clear` (which might be done to indicate that it is the player's turn to
-move in a multiplayer game):
+Boolean mode of the API by calling `navigator.setClientBadge` without an
+argument, and `navigator.clearClientBadge` (which might be done to indicate that
+it is the player's turn to move in a multiplayer game):
 
 ```js
 if (myTurn())
-  Badge.set();
+  navigator.setClientBadge();
 else
-  Badge.clear();
+  navigator.clearClientBadge();
 ```
 
-The reason we are considering both the "document" and "handle" contexts in the
-same API, and not two separate badging APIs, is so that in the common case,
-developers can just set a badge for the origin and have it show up in whatever
-places the user agent wants to show it. However, if you just want to badge a
-specific set of URLs, and not the whole site, use the `scope` option (which
-might be done if you have a different "unread count" on each page):
+All of the above set the badge *only* on the current document (e.g., in the page
+tab), and won't affect any other documents, even those at the same URL. To set a
+badge that applies to the current app:
 
 ```js
-Badge.set(getUnreadCount(location.pathname), {scope: location});
+navigator.setAppBadge(getUnreadCount());
 ```
 
-The scope is a URL prefix; the badge is applied on all pages whose URL [starts
-with](https://www.w3.org/TR/appmanifest/#dfn-within-scope) that prefix.
-
-More advanced examples are given in a [separate document](docs/examples.md).
-
-## Badge scope
-
-This section explains the difference between badging a *set of URLs* versus
-badging a *document*.
+As above, a value of 0 clears the badge and `navigator.clearAppBadge` can also
+explicitly clear the badge for this origin. The effects of the app API are
```

Comment:
"origin" could be read as it leaks outside its scope, but it really just relates to the installed web application. 
 
```suggestion
explicitly clear the badge for this installed web application. The effects of the app API are
```